### ISSUE TITLE
any use case for `utilsScript`?

### ISSUE THREAD

**Issue by neekey** [patw0929/react-intl-tel-input#175]
Hi, Thanks for creating such a great component!

I just wonder if it is really necessary to load the `libphonenumber.js` using `useScript`. I highly suggest that the `libphonenumber` should just be used as direct dependence, it's just easier to use.

The user cases that I can come up with that anyone wants to use a different util Script are:

- (1) They have a more performant utils function to provide these utils
- (2) The logic for validating numbers or other things related to international numbers are changed

For situation (1), instead of injecting a different library, it will be easier and more straightforward just providing utils function hooks, like `customIsNumberValid()`, `customNumberParse()`, which will decouple your library and the external code.

If at the end the so called customUtilsFunction becomes an convention, you can just replace `libphonenumber` with that library.

For situation (2), if it changes, I think `libphonenumber` will just follow up and release a new version for it, you just need to update the version.

In conclusion, `utilsScript` is now just an obstacle standing in most of your users' way by trying to cover an unknow user case.

Let me know if I miss something : )

**Comment by dackmin** [comment]
What if I don't want to require a 500+kbs library inside my project and use a smaller one, like https://github.com/catamphetamine/libphonenumber-js ?

**Comment by nutboltu** [comment]
`utilsScript` is removed in v6.0.0. #245
